go/worker/storage/committee/node.go

```diff
@@ -40,6 +42,42 @@ var (
 
 	// ErrNonLocalBackend is the error returned when the storage backend doesn't implement the LocalBackend interface.
 	ErrNonLocalBackend = errors.New("storage: storage backend doesn't support local storage")
+
+	storageWorkerLastFullRound = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "oasis_worker_storage_full_round",
+			Help: "The last round that was fully synced and finalized.",
+		},
+		[]string{"round"},
+	)
+
+	storageWorkerLastSyncedRound = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "oasis_worker_storage_synced_round",
+			Help: "The last round that was synced but not yet finalized.",
+		},
+		[]string{"round"},
+	)
+
+	storageWorkerLastPendingRound = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "oasis_worker_storage_pending_round",
+			Help: "The last round that is in-flight for syncing.",
+		},
+		[]string{"round"},
+	)
+
+	storageWorkerCollectors = []prometheus.Collector{
+		storageWorkerLastFullRound,
+		storageWorkerLastSyncedRound,
+		storageWorkerLastPendingRound,
+	}
+
+	labelLastFullRound    = prometheus.Labels{"round": "last_full"}
```

Comment:
Can you explain the role of these labels? Shouldn't you instead have labels for things like the runtime ID? See:
* https://github.com/oasisprotocol/oasis-core/blob/0fd92a9d9f0a980c68bc8392e59e1ea8aa80074a/go/worker/compute/executor/committee/node.go#L69
* https://github.com/oasisprotocol/oasis-core/blob/0fd92a9d9f0a980c68bc8392e59e1ea8aa80074a/go/worker/compute/executor/committee/node.go#L187-L191